assets/js/main.js

```diff
@@ -78,6 +95,12 @@ $(document).ready(function () {
       window.whatIsStargatePlayer.stopVideo();
     }
   });
+
+  $('form[newsletter-form]').on('submit', function(event) {
+    event.preventDefault();
+
+    // TODO: add functionality to the newsletter form
```

Comment:
When will this be added? Does this mean the form does not accpt inputs??

Reply:
I just removed it. All the newsletter form functionality was moved to the [forms.js](https://github.com/stargate/website/pull/29/files#diff-6691c402018576fc349ee55dbe5ba690b975d26cbdc2baa68637e7fcbde1509e) file